setup/homebrew.sh

```diff
@@ -7,15 +7,16 @@ command -v xcode-select >/dev/null 2>&1 || {
 }
 
 # Install Homebrew if it is not currently available.
-command -v brew >/dev/null 2>&1 || {
+command -v xcode-selectcommand -v brew >/dev/null 2>&1 || {
   echo >&2 "Installing Homebrew"; \
   /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)";
 }
 
 # Clean up all the things.
 brew update
 brew upgrade
-brew prune
+# JFV:brew prune is ddepricated and now runs as part of cleanup
+# brew prune
```

Comment:
This is going to be a hard habit for you to break. Everything is under version control, so there is no need to explain your changes within a file or to keep depricated code. Commented out code is pretty frowned upon now days as no one trusts its or plans to use it.

Instead, just make the change and commit the change with a git commit message. `git commit -m "Removing depricated brew prune as it now runs in cleanup."`

Of course it's too late to do that now. But in the future, small commits with meaningful messages are very helpful to understand the history of a file.